### ISSUE TITLE
JTextField keeps text selected after focus lost

### ISSUE THREAD

**Issue by FoxyBOA** [weisJ/darklaf#57]
Version 1.3.3.6

Either selected through `JTextField .selectAll()` method or manually selected text keep selected after JTextField lost focus. The only way I found to unselect focus  it to add a listener to JTextField with `JTextField..setCaretPosition(0)` on focustLost event.

Is it a bug or expected behavior?
```
class TextFieldExample {
    public static void main(String args[]) {
        LafManager.install();
        
        JFrame f = new JFrame("TextField Example");
        JTextField t1, t2;
        t1 = new JTextField("Selected Text");
        t1.setBounds(50, 100, 200, 30);
        t1.selectAll();
        
        t2 = new JTextField("Unselected text");
        t2.setBounds(50, 150, 200, 30);

        f.add(t1);
        f.add(t2);
        f.setSize(400, 400);
        f.setLayout(null);
        f.setVisible(true);
    }
}
```

**Comment by weisJ** [comment]
Thank you for reporting. This has been fixed and will be available in the next release.
If you still want to keep the selection when focus is lost you can achieve it by setting
````java
textField.putClientProperty("JTextField.keepSelectionOnFocusLost", true);
````

**Comment by FoxyBOA** [comment]
Thank you for the quick reply.

Please update your workaround code to

`textField.putClientProperty("JTextField.keepSelectionOnFocusLost", false)`;

**Comment by weisJ** [comment]
AM I missing something here? It should definitely be set to `true`. Losing focus should be the default behaviour. Or are you asking me to change the behaviour of the flag?

**Comment by weisJ** [comment]
Fix available with [`1.4.0.0`](https://github.com/weisJ/darklaf/releases/tag/v1.4.0.0)
